inference-engine/thirdparty/clDNN/src/impls/ocl/register.cpp

```diff
@@ -63,7 +63,6 @@ void register_implementations() {
     REGISTER_OCL(space_to_depth);
     REGISTER_OCL(strided_slice);
     REGISTER_OCL(tile);
-    REGISTER_OCL(fused_conv_eltwise);
```

Comment:
[random spot] There is a `"FUSED_CONV_ELTWISE": ` section in cache.json file whish should be removed

Reply:
I've removed fused_conv_eltwise_* from cache.json in manual. Isn't there any script to recreate cache.json?